assets/package.json

```diff
@@ -0,0 +1,140 @@
+{
+	"scripts": {
+		"build": "npm run build:production",
+		"prebuild:dev": "npm run remove-dist",
+		"build:dev": "webpack --mode=development --devtool cheap-module-eval-source-map --output-pathinfo",
+		"prebuild:production": "npm run remove-dist",
+		"build:production": "webpack --mode=production",
+		"build:test": "npm run build:production -- --include-tests",
+		"watch": "npm run build:dev -- --watch",
+		"remove-dist": "rimraf ../dist"
+	},
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
+	],
+	"dependencies": {
+		"@lit-labs/react": "^1.2.0",
+		"@material-ui/core": "^4.11.4",
+		"@material/button": "^2.3.0",
+		"@material/checkbox": "^2.3.0",
+		"@material/dialog": "^2.3.0",
+		"@material/form-field": "^2.3.0",
+		"@material/layout-grid": "^0.41.0",
+		"@material/linear-progress": "^1.1.0",
+		"@material/list": "^2.3.0",
+		"@material/menu": "^2.3.0",
+		"@material/radio": "^2.3.0",
+		"@material/react-card": "^0.15.0",
+		"@material/react-chips": "^0.15.0",
+		"@material/react-dialog": "^0.15.0",
+		"@material/react-select": "^0.15.0",
+		"@material/react-tab-bar": "^0.15.0",
+		"@material/react-text-field": "^0.15.0",
+		"@material/ripple": "^2.3.0",
+		"@material/select": "^2.3.1",
+		"@material/switch": "^2.3.0",
+		"@material/textfield": "^2.3.1",
+		"@material/theme": "^1.1.0",
+		"@material/web": "^1.0.0-pre.9",
+		"@popperjs/core": "^2.9.2",
+		"@reach/combobox": "^0.10.5",
+		"@react-hook/merged-ref": "^1.3.0",
+		"@react-hook/window-size": "^3.0.7",
+		"@wordpress/api-fetch": "^3.18.0",
+		"@wordpress/compose": "^3.25.3",
+		"@wordpress/data": "^4.27.3",
+		"@wordpress/dom-ready": "^2.10.0",
+		"@wordpress/element": "^2.20.3",
+		"@wordpress/i18n": "^3.14.0",
+		"@wordpress/icons": "^2.9.0",
+		"@wordpress/keycodes": "^3.2.4",
+		"@wordpress/url": "^3.1.1",
+		"accessible-autocomplete": "^2.0.3",
+		"classnames": "^2.2.6",
+		"clipboard-copy": "^3.1.0",
+		"compare-versions": "^3.6.0",
+		"dompurify": "^2.5.8",
+		"element-closest": "^3.0.2",
+		"focus-trap-react": "^10.0.0",
+		"formdata-polyfill": "^3.0.20",
+		"immer": "^9.0.15",
+		"intersection-observer": "^0.12.0",
+		"invariant": "^2.2.4",
+		"just-detect-adblock": "^1.1.0",
+		"lodash": "^4.17.21",
+		"md5": "^2.2.1",
+		"memize": "^1.1.0",
+		"polyfill-library": "^3.111.0",
+		"promise-polyfill": "^8.1.3",
+		"prop-types": "^15.7.2",
+		"psl": "^1.8.0",
+		"punycode": "^2.1.1",
+		"query-string": "^7.0.1",
+		"react": "^16.14.0",
+		"react-dom": "^16.14.0",
+		"react-google-charts": "^3.0.15",
+		"react-joyride": "^2.9.3",
+		"react-router-dom": "^5.2.0",
+		"react-use": "^15.3.4",
+		"react-use-observer": "^2.2.4",
+		"svgxuse": "^1.2.6",
+		"url-polyfill": "^1.1.9",
+		"use-memo-one": "^1.1.2",
+		"uuid": "^8.2.0",
+		"whatwg-fetch": "^3.2.0"
```

Comment:
These two are also not used anymore. We can delete them.

Reply:
LGTM, removed.